data/projects/implement_metrics_for_coala.md

```diff
@@ -7,28 +7,46 @@ them can be measured objectively by so-called
 offers a good base for the implementation or wrapping of algorithms that
 calculate these metrics from a piece of code.
 
+coala should have a module that can produce software metrics because software
+metrics are crucial when it comes to quality assurance. Source code metrics
+for quality assurance aims at writing code with less bugs inside and writing
+code for easy maintenance. Both lead to writing code as simple as possible
+with few elements in each unit. Picking right metrics is paramount and this
+project aims at implementing an API and a set of Bears for producing
+software metrics.
+
```

Comment:
I like what you added but feel like the first three paragraphs could be combined for more fluent reading. Some stuff is in there multiple times